pkg/controller/directvolumemigration/rsync.go

```diff
@@ -2162,14 +2162,19 @@ func (t *Task) reconcileRsyncOperationState(client compat.Client, req *rsyncClie
 				operation.CurrentAttempt += 1
 				// indicate that the operation is not yet completely failed, we will retry
 				currentStatus.pending = true
-				t.Log.Info("previous attempt of Rsync failed, created a new Rsync Pod", "pvc", operation, "attempt", operation.CurrentAttempt)
+				t.Log.Info("Previous attempt of Rsync failed, created a new Rsync Pod", "pvc", operation, "attempt", operation.CurrentAttempt)
 			} else {
 				operation.Failed = currentStatus.failed
 				operation.Succeeded = currentStatus.succeeded
 				if operation.IsComplete() {
 					t.Log.Info(
 						fmt.Sprintf("Rsync operation completed after %d attempts", operation.CurrentAttempt),
 						"pvc", operation, "failed", operation.Failed, "succeded", operation.Succeeded)
+				} else {
+					t.Log.Info("Rsync operation is still running. Waiting for completion",
```

Comment:
I also added this log message to give the user an idea of why the controller isn't proceeding.